packages/mco/components/mco-dashboard/data-policy/dr-dashboard-context.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { PrometheusResponse } from '@openshift-console/dynamic-plugin-sdk';
+import { ArgoApplicationSetResourceKind } from '../../../hooks';
+
+export const CSVStatusesContext = React.createContext<CSVStatusesContextType>(
+  {} as CSVStatusesContextType
+);
+
+export const DRResourcesContext = React.createContext<DRResourcesContextType>(
```

Comment:
```suggestion
export const DRDashboardContext = React.createContext<DRDashboardContextType>(
```

type DRDashboardContextType = {
  drResources: DRResourcesContextType;
  csvStatus: CSVStatusType
}

Reply:
that is intentional...
let's keep both separate... not all cards needs csvStatus... only those who need it will useContext for CSV... rest will not...

Reply:
You can maintain more than one prop in context, but more than one context for the same dashboard is not recommended.

Reply:
but its ok, we can keep it, if we start maintaining context per cards then it will turn into very complex.

other cards can still use the same context and they can extract the data that they want from the same context.

Reply:
Ref: https://github.com/stolostron/console/blob/main/frontend/src/lib/PluginContext.tsx

Reply:
single context will not work here... we need separate IMO... if csv output is changing we do not want to update those cards which are not using csv data at all...
this way there is separation... if csv data changes only those cards will get re-rendered that are using csv context and if drResources data is changed only that cards will get re-rendered which are using context of drresources...

Reply:
as we discussed, we are going keep the multiple context here